app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -113,71 +113,91 @@ hr {
   width: 100%;
 }
 
-// nav background color
-.top-bar, .top-bar li, .top-bar ul.menu  {
-  // background: $light-blue;
-  // background: $white;
+.top-bar {
   background: $blue;
-}
-
-
-// nav links
-.top-bar, .top-bar li, .top-bar ul.menu, .top-bar a {
-  // color: $grey;
   color: $white;
-  font-weight: bold;
-}
 
-// .top-bar li:hover, .top-bar ul.menu:hover,
-.top-bar a:hover {
-  color: $light-grey;
-}
+  .top-bar-title {
+    padding: 0;
 
-.top-bar {
-  a.active[disabled] {
-    color: $dark-grey;
-    cursor: default;
+    @media only screen and (max-width: #{map-get($breakpoints,medium)}) {
+      width: 100%;
+    }
+    img {
+      display: inline-block;
+    }
   }
-}
 
-// dropdown menu arrow color
-.dropdown.menu li.is-dropdown-submenu-parent a::after  {
-  // border-color: $grey transparent transparent;
-  border-color: $white transparent transparent;
-}
+  .menu-hamburger {
+    @media only screen and (max-width: #{map-get($breakpoints,medium)}) {
+      padding: .4rem;
+    }
+    float: right;
+    .menu-text {
+      color: $off-white;
+    }
+  }
 
-// nav dropdown menu
-.top-bar ul.menu {
-  border :none;
-}
+  ul.menu {
+    background: $blue;
+    border: none;
+
+    li {
+      a {
+        color: $off-white;
+        font-weight: bold;
+  
+        &.active[disabled] {
+          color: lighten($blue, 18%);
```

Comment:
That sounds good -- I can create an issue for it if you haven't already.

We might be able to use a website like colr.org or similar to find a better color scheme.